scripts/set_xtensa_params.sh

```diff
@@ -99,25 +99,28 @@ case "$platform" in
 	;;
     rmb)
 	PLATFORM="rembrandt"
-	ARCH="xtensa"
 	XTENSA_CORE="LX7_HiFi5_PROD"
 	HOST="xtensa-rmb-elf"
 	TOOLCHAIN_VER="RI-2019.1-linux"
 	;;
     vangogh)
 	PLATFORM="vangogh"
-	ARCH="xtensa"
 	XTENSA_CORE="ACP_5_0_001_PROD"
 	HOST="xtensa-vangogh-elf"
 	TOOLCHAIN_VER="RI-2019.1-linux"
 	;;
     acp_6_3)
 	PLATFORM="acp_6_3"
-	ARCH="xtensa"
 	XTENSA_CORE="ACP_6_3_HiFi5_PROD_Linux"
 	HOST="xtensa-acp_6_3-elf"
 	TOOLCHAIN_VER="RI-2021.6-linux"
 	;;
+	acp_7_0)
+ 	PLATFORM="acp_7_0"
```

Comment:
You can also remove PLATFORM now (and solve the small, new, corresponding git conflict)

Reply:
But, `PLATFORM` is being used in `xtensa-build-all.sh` at [here].(https://github.com/thesofproject/sof/blob/f383b0ffab96ea336448788266e797cc8ceb4505/scripts/xtensa-build-all.sh#L258)

Reply:
Done